### PR TITLE
Add DEGIRO & AnycoinDirect

### PR DESCRIPTION
**Pull request by remcoverstappen**

### CODE REVIEW

File: _data/cryptocurrencies.yml [hunk 1]
```diff
@@ -5,6 +5,13 @@ websites:
       tfa: Yes
       software: Yes
       doc: https://anxpro.com/pages/security
+
```
Comment (Carlgo11) [inline _data/cryptocurrencies.yml]:
```suggestion

```

File: _data/cryptocurrencies.yml [hunk 2]
```diff
@@ -5,6 +5,13 @@ websites:
       tfa: Yes
       software: Yes
       doc: https://anxpro.com/pages/security
+      
+    - name: AnycoinDirect
+      url: https://anycoindirect.eu
+      img: anycoindirect.png
+      tfa: Yes
+      software: Yes
+      doc: https://support.anycoindirect.eu/hc/en-gb/articles/115001423191-How-to-add-delete-2FA-on-my-Anycoin-Direct-account-
```
Comment (Carlgo11) [inline _data/cryptocurrencies.yml]:
```suggestion
      doc: https://support.anycoindirect.eu/hc/en-gb/articles/115001423191
```

File: _data/finance.yml [hunk 3]
```diff
@@ -46,6 +46,12 @@ websites:
       tfa: Yes
       sms: Yes
       phone: Yes
+
```
Comment (Carlgo11) [inline _data/finance.yml]:
```suggestion

```

File: _data/finance.yml [hunk 4]
```diff
@@ -46,6 +46,12 @@ websites:
       tfa: Yes
       sms: Yes
       phone: Yes
+      
+    - name: DEGIRO
+      url: https://www.degiro.nl
+      img: degiro.png
+      tfa: Yes
+      software: Yes
```
Comment (Carlgo11) [inline _data/finance.yml:55]: Can you provide some form of documentation on how to enable 2FA on DEGIRO?
If not, could you please attach a screenshot of the login process or 2FA setup process? (With personal information blurred out)

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
We usually only allow one site addition per PR but since this is your first pull request I guess it'll work.

I've made some changes, please merge those.

Also, here are updated versions of the logos, please use those instead:
anycoin
degiro

File: _data/finance.yml (hunk 4 above)
Comment (remcoverstappen) [reply _data/finance.yml:55]: Unfortunately DEGIRO does not provide documentation for 2FA. I've attached a screenshot of the setup and logon page (pages are only provided in Dutch).

**Review by Carlgo11 — DISMISSED** [review]

**Review by stephengroat — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**